### PR TITLE
Remove apiclientlib

### PR DESCRIPTION
**Pull request by protoism**
plugin apiclientlib is used only for percent encoding.

I replaced [google-http-java-client](https://github.com/googleapis/google-http-java-client/tree/main)'s 
PercentEncoder with java's URLEncoder

So... apiclientlib can be removed

### CODE REVIEW

**Comment by niloc132** [comment]
I think you meant to target this to main?

File: plugins/com.gwtplugins.gdt.eclipse.suite.update/src/com/google/gdt/eclipse/suite/update/usage/AnalyticsPingManager.java [hunk 1]
```diff
@@ -169,7 +168,7 @@ private static String getParametersString(Map<String, String> parametersMap) {
       }
       resultBuilder.append(entry.getKey());
       resultBuilder.append('=');
-      resultBuilder.append(ESCAPER.escape(entry.getValue()));
+      resultBuilder.append(URLEncoder.encode(entry.getValue(), Charsets.UTF_8));
```
Comment (niloc132) [inline plugins/com.gwtplugins.gdt.eclipse.suite.update/src/com/google/gdt/eclipse/suite/update/usage/AnalyticsPingManager.java:171]: could also be `java.nio.charset.StandardCharsets.UTF_8`

Comment (protoism) [reply plugins/com.gwtplugins.gdt.eclipse.suite.update/src/com/google/gdt/eclipse/suite/update/usage/AnalyticsPingManager.java:171]: Yes!

**Comment by protoism** [comment]
> I think you meant to target this to main?
After merging [feature/fix_eclipse_compilation](https://github.com/gwt-plugins/gwt-eclipse-plugin/tree/feature/fix_eclipse_compilation) I'll rebase it.
3 PRs one on top of each other.
Simplest way to test all of them together

**Pull request merged**